contracts/strategies/StrategyProxy.sol

```diff
@@ -44,7 +44,8 @@ contract StrategyProxy {
     }
 
     function lock() external {
-        proxy.increaseAmount(IERC20(crv).balanceOf(address(proxy)));
+        uint256 amount = IERC20(crv).balanceOf(address(proxy));
+        if (amount > 0) proxy.increaseAmount(amount);
```

Comment:
Prevents bricking down the line in the Curve contracts:
https://github.com/curvefi/curve-dao-contracts/blob/ba72347b8a6dced53bf80e6804fa5448eb6a753f/contracts/VotingEscrow.vy#L441